src/logservice/logminer/ob_log_miner_logger.cpp

```diff
@@ -79,8 +81,16 @@ int LogMinerLogger::log_progress(int64_t record_num, int64_t current_ts, int64_t
       nls_format, 0, time_buf, sizeof(time_buf), pos))) {
     LOG_WARN("datetime to string failed", K(current_ts), K(LOGMINER_TZ.get_tz_info()));
   } else {
-    fprintf(stdout, "\r%s %s %5.1lf%%, written records: %-20jd", time_buf, pb_buf,
-        progress, record_num);
+    int64_t current_time = ObTimeUtility::current_time();
+    int64_t speed = 0;
+    if (begin_ts_ == 0) {
+      begin_ts_ = current_time;
+    }
+    if (current_time - begin_ts_ > 0) {
+      speed = record_num * 1000 * 1000 / (current_time - begin_ts_);
+    }
+    fprintf(stdout, "\r%s %s %5.1lf%%, written records: %jd, speed: %jd/s", time_buf, pb_buf,
```

Comment:
Actually, the rate here is the average rate over time, not the current rate. Which do you think is more appropriate? Personally, I feel that command line tools such as `wget`, which display rates, prefer to use the current rate.

Reply:
you can have a look at `src/logservice/libobcdc/src/ob_log_trans_stat_mgr.h::TransRpsStatInfo` about how to calculate `RPS`

Reply:
speed -> rps, maybe more appropriate?